content/blog/api-console-for-data-services-cloud-console.md

```diff
@@ -0,0 +1,196 @@
+
+This blog will walk through the essential steps required to exercise or experiment with the DSCC REST API.
+
+### Authentication Process to Obtain the Access Token
+
+The DSCC public API relies on an OAuth 2.0 third party authorization framework on behalf of the resource owner (HPE GreenLake Console's user) for security. The user starts by logging and authenticating into HPE GreenLake Console, which is authenticated by the Identity Provider (validated through username, password, or MFA). Using the API Gateway menu in GreenLake Console, customer registers their client application to obtain the OAuth 2.0 API Client Credentials (Client ID and Client Secret). This association allows user to obtain the access-token from the menu, and user can then use the access-token into any REST API call inside the token bearer field. This action allows any client application or script to perform any HTTPS method to the correct instance of DSCC.
+
+![client-credential application flow](/img/greenlake-api-access-flow.png "obtain client-id and client-secret")
+
+The access-token have a limited lifetime (about 720 seconds or 2 hours). Once it expires, the client application must use the obtained Client-ID and Client-Secret to generate a new access-token. One indication of the expiration of access-token, the request to DSCC API will return response error: '401 Unauthorized HTTP.'  If the client application generates new access token prior to it expires, then the current access-token will be invalidated or treated as not authorized. 
+
+Additionally, user can also change the client-secret to update the authorization when authorized user lost the client-secret or the secret is compromised. 
+
+And lastly, when access to the API must be disabled, user can delete the API client credential associated with client-id and client-secret in the API Gateway.
+
+![Access API process](/img/user-guide-for-authorization.png "Process to authenticate and to obtain secure access ")
+
+## Accessing the API Gateway Console
+
+To access the API console, user must login into the [GreenLake Console,](https://common.cloud.hpe.com) deployed the DSCC to the intended region, and onboarded a storage array (Alletra, Nimble, or Primera) into the organization that is associated with the user account. The user must have the role that is required to perform the intended operation at the instance of DSCC where the storage is deployed. e.g. User must have volume management capability in the Data Ops Management to create a storage volume in US West. For more information about the role based access control, please take a look at the [HPE GreenLake User Guide](https://support.hpe.com/hpesc/public/docDisplay?docId=ccs-help_en_us)
+
+The **API** **Gateway** menu is available inside **Manage** menu. From **Greenlake Console** click on **Menu** to access this **Manage** Menu.
```

Comment:
Change to:
The API Gateway menu is available inside **the** Manage menu. From **HPE GreenLake** Console**,** click on Menu to access this Manage menu.